test/load/allocation/grpc/fixed.txt

```diff
@@ -1,23 +0,0 @@
-# Copyright 2022 Google LLC All Rights Reserved.
```

Comment:
We should also move this file under `test/load/allocation`

Reply:
The file is ignored in the commit due to gitignore. Do I update the gitignore and push these files ?

Reply:
The gitignore actually has statement to include those two files i.e.`!fixed.txt` and `!variable.txt`, but the order seems incorrect. I think to fix it we should put those two statements after `*.txt`, since what `!` does is "_any matching file excluded by a **previous** pattern will become included again_." [doc](https://git-scm.com/docs/gitignore).

Reply:
Thanks, I've updated the PR.